src/query/functions-v2/src/scalars/array.rs

```diff
@@ -81,4 +87,53 @@ pub fn register(registry: &mut FunctionRegistry) {
             }),
         }))
     });
+
+    registry.register_passthrough_nullable_1_arg::<EmptyArrayType, UInt64Type, _, _>(
+        "length",
+        FunctionProperty::default(),
+        |_| None,
+        vectorize_with_builder_1_arg::<EmptyArrayType, UInt64Type>(|_arr, output, _| {
+            output.push(0);
+            Ok(())
+        }),
+    );
+
+    registry.register_passthrough_nullable_1_arg::<ArrayType<GenericType<0>>, UInt64Type, _, _>(
+        "length",
+        FunctionProperty::default(),
+        |_| None,
+        vectorize_with_builder_1_arg::<ArrayType<GenericType<0>>, UInt64Type>(|arr, output, _| {
+            output.push(arr.len() as u64);
+            Ok(())
+        }),
+    );
+
+    registry.register_2_arg_core::<EmptyArrayType, UInt64Type, NullType, _, _>(
+        "get",
+        FunctionProperty::default(),
+        |_, _| Some(()),
+        |_, _, _| Ok(Value::Scalar(())),
+    );
+
+    registry.register_2_arg_core::<NullableType<EmptyArrayType>, NullableType<UInt64Type>, NullType, _, _>(
```

Comment:
duplicated with the previous `get EmptyArrayType`, just keep one, it's trivial for performance.

Reply:
ok